cpp/examples/strings/common.hpp

```diff
@@ -110,7 +110,7 @@ int main(int argc, char const** argv)
 
   std::chrono::duration<double> elapsed = std::chrono::steady_clock::now() - st;
   std::cout << "Wall time: " << elapsed.count() << " seconds\n";
-  std::cout << "Output size " << result->view().child(1).size() << " bytes\n";
+  std::cout << "Output size " << result->view().size() << " bytes\n";
```

Comment:
The column size here is not the number of bytes.
```suggestion
  auto const scv = cudf::strings_column_view(result->view());
  std::cout << "Output size " << scv.chars_size(rmm::cuda_stream_default) << " bytes\n";
```

Reply:
Thanks for the correction. Added to the PR